Gems/Multiplayer/Code/Include/Multiplayer/Components/NetworkHierarchyBus.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+
+namespace Multiplayer
+{
+    class NetworkHierarchyNotifications
+        : public AZ::ComponentBus
+    {
+    public:
+        //! Called when a hierarchy has been updated (a child added or removed, etc.)
+        virtual void OnNetworkHierarchyUpdated([[maybe_unused]] const AZ::EntityId& rootEntityId) {}
+
+        //! Called when an entity has left a hierarchy
+        virtual void OnLeavingNetworkHierarchy() {}
```

Comment:
Nit: Naming here is a bit inconsistent. Recommend changing the second function to OnNetworkHiearchyLeave.

Reply:
Sounds good, fixing.